includes/Classifai/Watson/Linker.php

```diff
@@ -73,9 +85,9 @@ public function link( $post_id, $output, $options = [] ) {
 	 *
 	 * @param int   $post_id The id of the post to link
 	 * @param array $categories The list of categories to link
-	 * @return void
+	 * @param bool  $return_terms Whether to return the terms to link or not
```

Comment:
I would suggest we change the logic here a bit. I think it's useful to always return the terms, I'd suggest instead this variable should determine whether we actually set the terms or not. So always return term data but we only set the terms here if this is `true` (which would be default).